compiler/passes/ResolveScope.cpp

```diff
@@ -663,6 +663,18 @@ void ResolveScope::getFields(const char* fieldName,
       INT_ASSERT(false);
     }
   }
+
+  if (symbols.size() > 0) {
```

Comment:
I double checked and this implementation only impacts use statements, not normal scope resolution